lib/matplotlib/axes/_axes.py

```diff
@@ -2902,6 +2902,11 @@ def pie(self, x, explode=None, labels=None, colors=None,
         shadow : bool, default: False
             Draw a shadow beneath the pie.
 
+        normalize: None or bool, default: None
+            When *True*, always make a full pie by normalizing x so that sum(x) == 1.
+            When *False*, make a partial pie.
+            When *None*, gives the current behavior and warns if sum(x) < 1.
```

Comment:
It's not obvious what the "current behavior is". I would move the whole paragraph about normalization from above into the parameter description (maybe needs a bit of rewriting).

Also, please mention here that *None* is deprecated.